leaf-ui/rappid-extensions/AnalysisInspector.js

```diff
@@ -77,7 +77,6 @@ var AnalysisInspector = Backbone.View.extend({
 		// These functions are used to communicate between analysisInspector and Main.js
 		this._analysisFunctions = analysisFunctions;
 		this.$el.html(_.template(this.template)());
-		$('head').append('<script src="./scripts/js-objects/analysis.js"></script>');
```

Comment:
Why is this script no longer needed?
Is it called from somewhere else?

Reply:
This script is still needed. It is called on analysis.html that pops up for "get next possible states" analysis. I had to remove from there because for some reason it started to crash when I tried to execute locally. But I need to figure out how to add it again some how without crashing...
I will fix this before making the pull request again